examples/digits_dann_lightn/main.py

```diff
@@ -3,24 +3,27 @@
 Reference: https://github.com/thuml/CDAN/blob/master/pytorch/train_image.py
 """
 
-import os
 import argparse
+
 # import warnings
 # import sys
 import logging
+import os
 
-# sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
-
-from kale.utils.csv_logger import setup_logger  # np error if move this to later, not sure why
```

Comment:
So this error doesn't exist after moving, does it?

Reply:
Checked. No error but a Numpy warning which seems not related to this change. That's fine.
`UserWarning: The given NumPy array is not writeable, and PyTorch does not support non-writeable tensors.`

Reply:
> So this error doesn't exist after moving, does it?

The previous wrapping was due to length 88. This line is <120 so no problem after setting black to length 120. That being said, you raised a very good point. Now the imports are resorted. That kind of fix will not be allowed anymore. I will remove the comment. Glad to see you have tested that there is no error but just a numpy warning. Many thanks! Peer review really improves quality!

Reply:
Sorry to unresolve this, I find it is okay on my pc with Win10, but it raises an error on HPC. After reverting it, the error is solved.

```
Traceback (most recent call last):
  File "main.py", line 13, in <module>
    from examples.action_dann_lightn.config import get_cfg_defaults
ModuleNotFoundError: No module named 'examples'
All is done!
```

Reply:
@XianyuanLiu in your examples, you could import those local modules as `from config import get_cfg_defaults`, as in digit_dann_lightn We can talk tomorrow if this is till an issue. Thanks.

Reply:
Solved by `pip install .`. Many thanks 💯 
FTR, [set & save environment variable in conda envs](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables).

Reply:
What is your second line of reply on "set & save environment variable in conda envs." for? I can't get it without context?

Reply:
Sorry. It is a record. Before you told me to use `pip install .`, I have followed the link to add `pykale` root path to the environment variable in my conda env and solved the problem. It is not the best option but may be useful in some other situations.